instances/atlas/brainAtlasVersion/WHSSDatlas_v3.jsonld

```diff
@@ -380,20 +380,24 @@
 		],
 		"ontologyIdentifier": null
 	},
-  "homepage": null,
-  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the first publication (Papp et al., 2014) along with the publication for the atlas version you have used (see Related publications below).",
+  "homepage": "https://www.nitrc.org/projects/whs-sd-atlas/",
+  "howToCite": "Please refer to the atlas by its RRID: SCR_017124, and cite the publications for WHS rat brain atlas v1 (Papp EA, Leergaard TB, Calabrese E, Johnson GA, Bjaalie JG (2014) Waxholm Space atlas of the Sprague Dawley rat brain. NeuroImage 97, 374-386. DOI: 10.1016/j.neuroimage.2014.04.001) and WHS rat brain atlas v3 (Osen KK, Imad J, Wennberg AE, Papp EA, Leergaard TB (2019) Waxholm Space atlas of the rat brain auditory system: Three-dimensional delineations based on structural and diffusion tensor magnetic resonance imaging. NeuroImage 199, 38-56. DOI: 10.1016/j.neuroimage.2019.05.016).",
   "isAlternativeVersionOf": null,
   "isNewVersionOf": {
     "@id": "https://openminds.ebrains.eu/instances/brainAtlasVersion/WHSSDatlas_v2"
   },
   "keyword": null,
   "license": {
-    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBySa4.0"
+    "@id": "https://openminds.ebrains.eu/instances/licenses/ccBy4.0"
   },
   "majorVersionIdentifier": null,
   "ontologyIdentifier": null,
   "otherContribution": null,
-  "relatedPublication": null,
+  "relatedPublication": [
+	{"@id": "https://doi.org/10.1016/j.neuroimage.2014.04.001"},

```

Comment:
please correct the indentation (2 spaces more for each list item)

Reply:
Fixed.